src/generator/www/js/main.js

```diff
@@ -0,0 +1,51 @@
+$(document).ready(function() {
+
+  var $email =$('#email');
+  var $name  =$('#name'); 
+  var $url  =$('#url');
+
+  $('#btn').click(function() { 
+
+  var form = document.querySelector('form');
+  var fd = new FormData(form);
+
+  var files ={
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 space characters but found 2.](https://www.codacy.com/app/triplez/open-event-webapp/file/2711063426/issues/source?bid=3233857&fileBranchId=3344140#l12)